src/components/tx-flow/flows/RecoveryAttempt/RecoveryAttemptReview.tsx

```diff
@@ -0,0 +1,93 @@
+import { type SyntheticEvent, useState, useContext, useCallback } from 'react'
+import { CircularProgress, CardActions, Button, Typography, Stack, Divider } from '@mui/material'
+import EthHashInfo from '@/components/common/EthHashInfo'
+import CheckWallet from '@/components/common/CheckWallet'
+import { Errors, trackError } from '@/services/exceptions'
+import { dispatchRecoveryExecution } from '@/features/recovery/services/recovery-sender'
+import useWallet from '@/hooks/wallets/useWallet'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import type { TransactionAddedEvent } from '@gnosis.pm/zodiac/dist/cjs/types/Delay'
+import ErrorMessage from '@/components/tx/ErrorMessage'
+import TxCard from '@/components/tx-flow/common/TxCard'
+import FieldsGrid from '@/components/tx/FieldsGrid'
+import { TxModalContext } from '../..'
+
+export type RecoveryAttemptReviewProps = {
+  params: {
+    args: TransactionAddedEvent.Log['args']
+    address: string
+  }
```

Comment:
It would be nice if we could avoid having nested parameters in the props, so this component would just be re-rendered only when it is necessary. So instead of passing the address and args inside params, you could pass them as main props, like: 

```suggestion

export type RecoveryAttemptReviewProps = {
    args: TransactionAddedEvent.Log['args']
    address: string
}
```

Reply:
It now takes (an even more nested) RecoveryQueueItem as a prop.